aries-site/src/data/structures/templates.js

```diff
@@ -231,4 +231,24 @@ export const templates = [
     sections: [],
     relatedContent: ['Page Layouts', 'Header', 'Navigation'],
   },
+  {
+    name: 'Table Customization',
+    available: false,
+    description: `Allows users to customize table column order and
```

Comment:
Wording suggestion, it wasn't immediately clear to me what "visibility" meant/referred to:

Allow users to customize which columns are visible in a table and in what order they appear.

Reply:
I like it. Thanks.